jsonrpc/eth_endpoint.go

```diff
@@ -35,20 +35,20 @@ func (e *Eth) GetBlockByNumber(number BlockNumber, full bool) (interface{}, erro
 		num = uint64(number)
 	}
 
-	block, ok := e.d.store.GetBlockByNumber(num, full)
+	block, ok := e.d.store.GetBlockByNumber(num, true)
 	if !ok {
-		return nil, fmt.Errorf("unable to get block by num %v", num)
+		return nil, nil
```

Comment:
Is there any specific reason why we're not returning an error here?

Reply:
I checked jsonrpc spec and tried with ethereum mainnet by infura. I just got null when block not found. So I think it's ok we don't need to return error.